api/proto/info/info.proto

```diff
@@ -229,6 +229,10 @@ message ZInfoAppInstance {
   string domainName = 3;
 }
 
+message AttestationInfo {
```

Comment:
What's the tradeoff between carrying this in the  device info message as opposed to in the attest message?

Also, should there be a last_attestation_success and/or next_attestation_attempt timestamp in the info? Or do we assume the controller can check internally?

Reply:
Actually @eriknordmark @cshari-zededa -- sorry for being dense, but what is the point of this Info message to begin with? I can't find a place where it plugs into the workflow described above. Care to elaborate?

Reply:
@rvs This is to store a (encrypted) copy of the volume encryption keys in Controller. The keys will be given back to the device(whenever it asks for it), only on a successful attestation.

 IOW, if the device goes for a reboot, and comes up with an unknown software, by default it will be able to unlock the volumes (since the keys are sealed with PCR values), and the only way for the device to get the keys back is by asking the Controller to share it, which it will get only after completing a successful attestation.  The keys themselves will be wrapped(aka encrypted) with a TPM private key, and key is published to Controller only in its encryption form - which means, for Controller it is just an opaque information, and only *this* device can make sense out of this.

More details at 
https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation#MeasuredBootandRemoteAttestation-SecurelyUpgradingEVESoftware

BTW, to address @eriknordmark 's point, I have moved this into /attest API endpoint, as it makes more sense to do it in the attest umbrella, rather than under general /info endpoint.